src/main/java/com/uber/jenkins/phabricator/conduit/DifferentialClient.java

```diff
@@ -51,33 +46,33 @@ public DifferentialClient(String diffID, LauncherFactory launcher, String condui
      * @param silent whether or not to trigger an email
      * @param action phabricator comment action, e.g. 'resign', 'reject', 'none'
      */
-    public JSONObject postComment(String revisionID, String message, boolean silent, String action) throws IOException, InterruptedException, ArcanistUsageException {
+    public JSONObject postComment(String revisionID, String message, boolean silent, String action) throws ConduitAPIException, IOException {
         Map params = new HashMap<String, String>();
         params.put("revision_id", revisionID);
         params.put("action", action);
         params.put("message", message);
-        params.put("silent", silent);
+        params.put("silent", Boolean.toString(silent));
 
         return this.callConduit("differential.createcomment", params);
     }
 
-    public JSONObject fetchDiff() throws ArcanistUsageException, IOException, InterruptedException {
+    public JSONObject fetchDiff() throws IOException, ConduitAPIException {
         Map params = new HashMap<String, String>();
-        params.put("ids", new String[]{this.diffID});
+        params.put("ids[]", diffID);
```

Comment:
Yea, the old one was JSON, and quite frankly we may need to end up switching back to JSON at some point to encode more complex data, but the "simple" conduit HTTP api encodes with URL-encoded POST form data.